types/humanize-string/index.d.ts

```diff
@@ -0,0 +1,9 @@
+// Type definitions for humanize-string 1.0
+// Project: https://github.com/sindresorhus/humanize-string/
+// Definitions by: Kevin Nacios <https://github.com/ragnarok56>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export = humanizeString;
+
+declare function humanizeString(text: string): string;
+declare namespace humanizeString {}
```

Comment:
I think this line can be removed, can you confirm?

Reply:
yup! good catch.  thought I needed it